apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-modal.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import LocationAutoCompleteTextbox from './location-autocomplete-textbox';
+import { CustomTextInput } from '../../../components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons';
+import { MullBackButton } from '../../../components';
+import { EventRestriction, ILocation } from '@mull/types';
+import { FormikConfig } from 'formik';
+
+export interface LocationAutocompleteModalProps {
+  formik: FormikConfig<{
+    activeRestriction: EventRestriction;
+    startDate: any;
+    endDate: any;
+    startTime: string;
+    endTime: string;
+    eventTitle: string;
+    description: string;
+    location: ILocation;
+    imageFile: string;
+  }>;
+}
+
+export default function LocationAutocompleteModal({
+  formik: { touched, setFieldValue, errors, values },
+}) {
```

Comment:
You're not using `LocationAutocompleteModalProps `, and losing out on Typescript goodness

See: https://github.com/formium/formik/issues/2023#issuecomment-596663900

Please create a `CreateEventForm` interface like the one below, put it in the `types` library, and set it as the type for the useFormik statement in `create-event.tsx`

In `LocationAutocompleteModalProps`, change the type of `formik` to `FormikContextType<CreateEventForm>`, as shown in the suggestion

```tsx
const formik = useFormik<CreateEventForm>({ // line 86
```

```suggestion
export interface LocationAutocompleteModalProps {
  formik: FormikContextType<CreateEventForm>;
}

export interface CreateEventForm {
  activeRestriction: EventRestriction;
  startDate: Date;
  endDate: Date;
  startTime: string;
  endTime: string;
  eventTitle: string;
  description: string;
  location: ILocation;
  imageFile: string;
}

export default function LocationAutocompleteModal({
  formik: { touched, setFieldValue, errors, values },
}: LocationAutocompleteModalProps) {
```